server/core/region_tree.go

```diff
@@ -63,7 +62,7 @@ func (t *regionTree) length() int {
 func (t *regionTree) update(region *metapb.Region) []*metapb.Region {
 	item := &regionItem{region: region}
 
-	// note that find() gets the first item equals or greater to the region
+	// note that find() gets the last item that is less or equal than the region.
```

Comment:
last > first

Reply:
_last_ means the last region in the regions that are less than or equal to the provided region. I think it makes sense.